src/test/resources/org/prebid/server/it/test-application.properties

```diff
@@ -150,8 +150,9 @@ adapters.edge226.enabled=true
 adapters.edge226.endpoint=http://localhost:8090/edge226-exchange
 adapters.emtv.enabled=true
 adapters.emtv.endpoint=http://localhost:8090/emtv-exchange
-adapters.emxdigital.enabled=true
-adapters.emxdigital.endpoint=http://localhost:8090/emx_digital-exchange
+adapters.emx_digital.enabled=true
+adapters.emx_digital.endpoint=http://localhost:8090/emx_digital-exchange
+adapters.emx_digital.aliases.cadent_aperture_mx.enabled=true
```

Comment:
Please, use original path that is present in .yaml configuration file.
`adapters.emxdigita. ...`